app/helpers/candidates/school_helper.rb

```diff
@@ -142,14 +140,18 @@ def start_request_link(school)
     end
   end
 
-private
-
   def split_to_list(content)
+    items = content&.split("\n")
```

Comment:
This now all looks great

My only nitpick is this line doing the processing needs to handle 'odd' scenarios - substituting with the below will mean blank lines are excluded rather than generating empty bullets

`content.split("\n").map(&:strip).compact`